attribute.go

```diff
@@ -22,8 +22,21 @@ type Attribute struct {
 
 	// An arbitrary payload which is specified by Type.
 	Data []byte
+
+	// Whether the attribute contains nested attributes
+	Nested bool
+
+	// Endianness of the payload
```

Comment:
`// Whether the attribute's data is in network byte order or native byte order.`, or something like that.